src/main/java/yapbot/exceptions/YapBotException.java

```diff
@@ -0,0 +1,19 @@
+package yapbot.exceptions;
+
+/**
+ * Custom Checked Exception for YapBot exceptions.
+ */
+public class YapBotException extends Exception {
+    private static final String PREFIX_LINE = "\n-------------------------------------------\n";
+    private static final String POSTFIX_LINE = "\n-------------------------------------------\n";
+
```

Comment:
I like that you added the lines to make the exception more prominent when printed out :+1: